### PR TITLE
Add User-Guide with some rough pages and content taken from forums.

### PR DESCRIPTION
**Pull request by DanVanAtta**
## Overview

Added a new header link 'User Guide' with a very basic index page that links
to further help/info pages.

The pages so far are some of the key rule and info pages grabbed from forums
with some quick and dirty formatting applied.

Further work would be great to reduce the "wall of text" problem (improve,
reduce and simplify the wording), improve formatting and perhaps add images.

CC/ @ron-murhammer  @prastle 

## Screenshot

Screenshot from 2019-06-06 21-26-29

## Additional Notes

- This is a rev1, kinda quick and dirty a bit, my hope is that we'll be able to iterate further on this content, add to it and make it look nicer.
- All content is existing content lifted from the forums, some formatting was applied.

### CODE REVIEW

File: user-guide/how-to-host.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+### Troubleshooting
+
+Computers often don't work like you want them to, here are some tricks that might come in handy 
+if something is not working for you:
+Try temporarily disabling your Firewall. If everything is working fine with your Firewall disabled,
+ you haven't setup your Firewall correctly. If it's still not working your Router isn't correctly 
+ forwarding the traffic to your PC.
+
+[canyouseeme.org](http://canyouseeme.org) is a website that helps you checking if your configuration is
```
Comment (RoiEXLab) [inline user-guide/how-to-host.md]:
```suggestion
[canyouseeme.org](https://canyouseeme.org) is a website that helps you checking if your configuration is 
```

File: user-guide/how-to-host.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+```
+@echo off
+SET PORT=3363
+SET BOT_NUMBER=25
+
+SET LOBBY_HOST=45.79.144.53
+SET LOBBY_PORT=3304
+
+java -server -Xmx320m -Djava.awt.headless=true -classpath bin/triplea.jar games.strategy.engine.framework.headlessGameServer.HeadlessGameServer -Ptriplea.game.host.console=false -Ptriplea.game= -Ptriplea.server=true -Ptriplea.port=%PORT% -Ptriplea.lobby.host=%LOBBY_HOST% -Ptriplea.lobby.port=%LOBBY_PORT% -Ptriplea.name=Bot%BOT_NUMBER%_Pras1 -Ptriplea.lobby.game.hostedBy=Bot%BOT_NUMBER%_Pras1 -Ptriplea.lobby.game.supportEmail=prastle7@hotmail.com -Ptriplea.lobby.game.comments="automated_host" -Ptriplea.lobby.game.reconnection=172800
```
Comment (RoiEXLab) [inline user-guide/how-to-host.md]: I just realized: We should probably remove @prastle 's email here

File: user-guide/lobby-rules.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+The admins do not care if it does not offend you and/or your friends. No bullying and/or 
+discrimination will be tolerated, that includes; Race, Religion, Sex, Age, Cultural Background, 
+Sexual Orientation. Use of foul language, discrimination and derogatory comments are not acceptable, 
+and are subject to admin disciplinary action. It is up to the admin to decide what is or is not 
+acceptable in regards to defamation, discrimination and spamming.
+
+* Due to the recent spat of BLATANT! racism in the lobby the TripleA Council has decided to increase 
+the punishment for these offences to an instant 2 week Ban on first offense!
+
+* No verbal abuse. Doing so violates rule #1 above.
```
Comment (RoiEXLab) [inline user-guide/lobby-rules.md]: \# might need escaping, just markdown things

File: user-guide/lobby-rules.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+up to and including being kicked from the game, or temporarily/permanently being banned. We have a 
+thread to lodge complaints, so use it and keep it out of the Lobby.
+
+* Please watch your language - there are underage children playing in the lobby, would you want 
+your younger brother/sister/cousin being exposed to the kind of language you're using? Keep it 
+clean when you get upset, take a breather before expressing your feelings in the Lobby and please 
+refrain from swearing.
+
+Our general policy in regards to breaking the rules is: Warn, Mute, Kick, Day-Ban, Perm Ban. 
+Depending on the seriousness of the issue, you may be insta-day-banned (remember rule #2.)
```
Comment (RoiEXLab) [inline user-guide/lobby-rules.md]: Same here with \# escaping

File: user-guide/lobby-rules.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+user, banning admin, or another admin involved, we ask that you do not post unnecessarily.
+
+### Lobby and Bot Rules
+
+Again the General Rules above apply to the Lobby as well to the bots. The bots are not private 
+rooms and the Admin are also moderating the bots. Admins have the right to take action upon you, 
+if you are found in breach of these rules.
+
+If you feel an admin has abused players or violated the rules, please PM another admin or post 
+here. Do not post a thread for flamebait. Be professional and take a screenshot and let another
```
Comment (RoiEXLab) [inline user-guide/lobby-rules.md]: Posting here? 😅

File: user-guide/lobby-rules.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+eventually in a ban from the server.
+
+The following usernames are explicit not allowed:
+
+- Names which contains vulgar language
+- Names with sexual or racial connotations
+- Names with a political or religious or ideological agenda
+- Names which impersonate another player or a mod and names which are made solely to harass, 
+troll or annoy another player or a mod.
+- Triplea is a wargame and the names of the protagonist of the different warscenarios hosted
```
Comment (RoiEXLab) [inline user-guide/lobby-rules.md]:
```suggestion
- TripleA is a wargame and the names of the protagonist of the different warscenarios hosted 
```

File: user-guide/how-to-host.md (hunk 2 above)
Comment (DanVanAtta) [reply user-guide/how-to-host.md]: Even worse than that, the recommended way to run a bot is via the 'run_bot.bat' script, to download that and modify. This is a great example of how we documented things multiple times and lead to documentation drift.

Comment (DanVanAtta) [reply user-guide/how-to-host.md]: I did a quick update of this: e9e061f

To update to be proper instructions of launching the bat file I'd like to leave for another iteration where we overhaul this document even more.

File: user-guide/lobby-rules.md (hunk 3 above)
Comment (DanVanAtta) [reply user-guide/lobby-rules.md]: Good consideration, happily only if the '#' is at start of line, seems to render okay:

Screenshot from 2019-06-08 13-10-49

File: user-guide/lobby-rules.md (hunk 5 above)
Comment (DanVanAtta) [reply user-guide/lobby-rules.md]: :) 
Updated: 9a514c4

File: user-guide/how-to-host.md (hunk 2 above)
Comment (DanVanAtta) [reply user-guide/how-to-host.md]: Note, used this script: https://github.com/triplea-game/triplea/blob/master/game-headless/scripts/run_bot.bat, for the list of updated params, they have been simplified since this was written. 
Again the proper solution is to give instructions to unzip the bot and then run the included bat script, these steps are redundant... Leaving for another day or person while I'm focusing on Mod toolbox right now.
